actors/user/src/main/java/org/sunbird/user/actors/UserManagementActor.java

```diff
@@ -148,6 +148,13 @@ private void updateUser(Request actorMessage) {
       userMap.put(JsonKey.UPDATED_BY, actorMessage.getContext().get(JsonKey.REQUESTED_BY));
     }
     Map<String, Object> requestMap = UserUtil.encryptUserData(userMap);
+    ProjectLogger.log("UserManagementActor:updateUser:user data successfully encrypted",LoggerEnum.INFO.name());
+    if(userMap.containsKey(JsonKey.RECOVERY_EMAIL)){
+      isRecoveryEmailEqualsPrimaryEmail(userDbRecord,(String) userMap.get(JsonKey.RECOVERY_EMAIL));
```

Comment:
method signature is not matching with implementation.
isRecoveryEmailEqualsPrimaryEmail should return with boolean then caller need to take action on that